.github/scripts/rebase-and-deploy-to-env.sh

```diff
@@ -29,14 +30,23 @@ if [[ "$TARGET_BRANCH" == "stable" ]]; then
   fi
 fi
 
+# If the deployment targets `prod`, then the feature must be on stable first.
+if [[ "$TARGET_BRANCH" == "prod" ]]; then
+  if [ $(git branch --contains $SHA_COMMIT | grep -c "stable") -eq 0 ]; then
+    printf "\U274C In order to deploy to prod branch the feature must be on stable branch first. Please deploy it there first, check that everything is running fine and then promote it to prod."
```

Comment:
I would also mention `dev` here to avoid the "cascading" error like
```
/deploy prod    >> Error: the commit is not on stage
/deploy stage   >> Error: the commit is not on dev 
...
```

Reply:
@danielezonca To make sure I follow, you mean update the message to provide better instructions e.g:

`In order to deploy to prod branch the feature must be on stable branch first. Please deploy the feature to dev and then to stable, check that everything is running fine and then, finally, promote it to prod.

Reply:
Yes 👍

Reply:
Done.